src/main/java/io/appform/dropwizard/sharding/DBShardingBundleBase.java

```diff
@@ -51,18 +52,13 @@
 import lombok.Getter;
 import lombok.extern.slf4j.Slf4j;
 import lombok.val;
+import lombok.var;
 import org.apache.commons.lang3.StringUtils;
 import org.hibernate.SessionFactory;
 import org.reflections.Reflections;
 
 import javax.persistence.Entity;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
-import java.util.Set;
+import java.util.*;
```

Comment:
Usually using wildcard imports is not considered a good practice. SonarQube also raises code smell for this

Reply:
Done